src/components/pages/devices/flyouts/deviceDelete/deviceDelete.js

```diff
@@ -129,11 +131,14 @@ export class DeviceDelete extends Component {
               </div>
             }
 
-            <SummarySection title={t('devices.flyouts.delete.summaryHeader')}>
-              <SummaryCount>{summaryCount}</SummaryCount>
-              <SectionDesc>{summaryMessage}</SectionDesc>
-              {this.state.isPending && <Indicator />}
-              {completedSuccessfully && <Svg className="summary-icon" path={svgs.apply} />}
+            <SummarySection>
+              <SectionHeader>{t('devices.flyouts.delete.summaryHeader')}</SectionHeader>
+              <SummaryBody>
+                <SummaryCount>{summaryCount}</SummaryCount>
+                <SectionDesc>{summaryMessage}</SectionDesc>
+                {this.state.isPending && <Indicator />}
```

Comment:
You should confirm with design but I believe the goal is the summary section indicators should be bar patterned.